keda/templates/12-keda-deployment.yaml

```
@@ -100,6 +100,10 @@ spec:
             - name: KEDA_HTTP_DEFAULT_TIMEOUT
               value: {{ .Values.http.timeout | quote }}
             {{- end }}
+            {{- if .Values.http.keepAlive.enabled }}
+            - name: KEDA_HTTP_DISABLE_KEEP_ALIVE
+              value: {{ .Values.http.keepAlive.enabled }}
+            {{- end }}
```

We can't do this as the current code has the opposite logic, when `.Values.http.keepAlive.enabled` is true, we disable it in the code.
I know that the code is already merged, so maybe we can handle this here adding some logic, for example:
```suggestion
            {{- if not( .Values.http.keepAlive.enabled) }}
            - name: KEDA_HTTP_DISABLE_KEEP_ALIVE
              value: "true"
            {{- end }}
```

And the same for the other manifest. WDYT?

---

@JorTurFer, done